test/test_utils/security.py

```diff
@@ -187,3 +262,280 @@ def get_ecr_scan_allowlist_path(image_uri):
     if "example" in image_uri:
         image_scan_allowlist_path = dockerfile_location + ".example.os_scan_allowlist.json"
     return image_scan_allowlist_path
+
+
+def _save_lists_in_s3(save_details, s3_bucket_name):
+    """
+    This method takes in a list of filenames and the data corresponding to each filename and stores it in 
+    the s3 bucket.
+
+    :param save_details: list[(string, list)], a lists of tuples wherein each tuple has a filename and the corresponding data.
+    :param s3_bucket_name: string, name of the s3 bucket
+    """
+    s3_client = boto3.client("s3")
+    for filename, data in save_details:
+        with open(filename, "w") as outfile:
+            json.dump(data, outfile, indent=4)
+        s3_client.upload_file(Filename=filename, Bucket=s3_bucket_name, Key=filename)
+
+
+def _botocore_resolver():
+    """
+    Get the DNS suffix for the given region.
+    :return: endpoint object
+    """
+    loader = botocore.loaders.create_loader()
+    return botocore.regions.EndpointResolver(loader.load_data("endpoints"))
+
+
+def get_ecr_registry(account, region):
+    """
+    Get prefix of ECR image URI
+    :param account: Account ID
+    :param region: region where ECR repo exists
+    :return: AWS ECR registry
+    """
+    endpoint_data = _botocore_resolver().construct_endpoint("ecr", region)
+    return "{}.dkr.{}".format(account, endpoint_data["hostname"])
```

Comment:
nit: I think this is more of an `ecr` util than a `security` util.

Reply:
Done, thanks!!